packages/swapper/package.json

```diff
@@ -15,5 +15,11 @@
     "clean": "rm -rf dist",
     "test": "jest test",
     "type-check": "tsc --project ./tsconfig.json --noEmit"
+  },
+  "dependencies": {
+    "@shapeshiftoss/asset-service": "^1.0.0",
```

Comment:
Is this just a temporary dependency until we get a shared types package going?

Reply:
Yes, this will be the common package when we create it.  All I'm getting is types right now